doozer/doozerlib/cli/scan_sources_konflux.py

```diff
@@ -640,9 +640,21 @@ def add_image_meta_change(self, meta: ImageMetadata, rebuild_hint: RebuildHint):
             self.add_assessment_reason(descendant_meta, RebuildHint(RebuildHintCode.ANCESTOR_CHANGING,
                                                                     f'Ancestor {meta.distgit_key} is changing'))
 
+    def is_image_enabled(self, image_name: str) -> bool:
+        image_meta = self.runtime.image_map[image_name]
```

Comment:
Would like to add a check to return False if the image_name is not in image_map.